module/userProjects/helper.js

```diff
@@ -3690,7 +3826,7 @@ function _entitiesMetaInformation(entitiesData) {
     return new Promise(async (resolve, reject) => {
         let entityInformation = []
         for ( let index = 0; index < entitiesData.length; index++ ) {
-            let entityHierarchy =  await userProfileService.getParentEntities( entitiesData[index]._id );
+            let entityHierarchy =  await userService.getParentEntities( entitiesData[index]._id );
```

Comment:
@Prajwal17Tunerlabs you dont need to call subird user service you can call entity service